test/functional/wallet_basic.py

```diff
@@ -309,6 +309,9 @@ def run_test(self):
         assert_equal(tx_obj['amount'], Decimal('-0.0001'))
 
         # General checks for errors from incorrect inputs
+        # This will raise an exception because the amount is negative
+        assert_raises_rpc_error(-3, "Amount out of range", self.nodes[0].sendtoaddress, self.nodes[2].getnewaddress(), "-1")
```

Comment:
now that I mention it, I'm a little shocked we don't have coverage on this?

Reply:
We do for the utility function via `createrawtransaction` tests.

Reply:
you mean `AmountFromValue`? I meant specifically for `sendtoaddress`. Anyways, a test is a win.